### ISSUE TITLE
Button color contrast fails when dark mode is on

### ISSUE THREAD

**Issue by seantwalsh** [mdlutz24/drupal-meeting-parser#8]
When using dark mode in Slack or at the OS level, the text color switches to white which does not pass on the yellow buttons. In order to maintain sufficient color contrast of the buttons and their text, both values should be set in js. Will make a PR soon.
